pyinfra/api/inventory.py

```diff
@@ -37,6 +37,10 @@ class Inventory(object):
         ssh_key: override SSH key filename
         ssh_key_password: override password for the SSH key
         ssh_password: override SSH password
+        winrm_username: override WINRM username
+        winrm_password: override WINRM pasword
+        winrm_port: override WINRM port
+        use_shell: override shell 
```

Comment:
I think this should be configured elsewhere; either in the deploy config, or  somehow in the command, so it can be changed during operations.

Reply:
Not really sure how to do that.